src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/sulu-content.js

```diff
@@ -366,6 +375,16 @@ define(function() {
          * Loads data if needed and start executing component handlers
          */
         app.components.before('initialize', function() {
+            // init options and translations
+            if (!!this.defaults) {
```

Comment:
this would be nice (: it would extend automatically default options and translations (if exists) ...